src/renderer/features/multisigs/model/__tests__/multisigs-model.test.ts

```diff
@@ -0,0 +1,134 @@
+import { allSettled, fork } from 'effector';
+
+import { networkModel } from '@entities/network';
+import {
+  ConnectionType,
+  AccountType,
+  ChainOptions,
+  WalletType,
+  SigningType,
+  CryptoType,
+  ChainType,
+} from '@shared/core';
+import { walletModel } from '@entities/wallet';
+import { multisigsModel } from '../multisigs-model';
+
+const signatories = [
+  {
+    accountId: '0x01',
+    address: 'F7NZ',
+  },
+  {
+    accountId: '0x02',
+    address: 'F7Pa',
+  },
+  {
+    accountId: '0x03',
+    address: 'F7XB',
+  },
+];
+
+jest.mock('@entities/multisig', () => ({
+  ...jest.requireActual('@entities/multisig'),
+  multisigService: {
```

Comment:
I tried with the following
```ts
jest.spyOn(multisigService, 'filterMultisigsAccounts').mockResolvedValue([
  {
    accountId: '0x00',
    threshold: 2,
    signatories: ['0x01', '0x02', '0x03'],
  },
]);
```

but this doesn't work, as in when I print the result of `multisigService.filterMultisigsAccounts` in the model, it never shows the log. It's called for sure because if I put a log before the call, this is printed.